src/Clapi/TextSerialisation.hs

```diff
@@ -26,33 +26,33 @@ cvBuilder (ClEnum i) = fromShow i
 tdTaggedBuilder :: TaggedData e a -> (a -> Builder) -> a -> Builder
 tdTaggedBuilder td bdr a = fromChar (tdInstanceToTag td $ a) <> bdr a
 
-msgBuilder :: Message -> Builder
-msgBuilder msg = case msg of
-    (MsgSet {}) -> valB 's' valSubs msg
-    (MsgAdd {}) -> valB 'a' valSubs msg
-    (MsgRemove {}) -> valB 'r' noSubs msg
+msgBuilder :: DataUpdateMessage -> Builder
+msgBuilder = tdTaggedBuilder Wire.dumtTaggedData dumBuilder
   where
+    dumBuilder msg = case msg of
+        (UMsgSet {}) -> tab valSubs msg
+        (UMsgAdd {}) -> tab valSubs msg
+        (UMsgRemove {}) -> tab noSubs msg
     tb (Time s f) = (fromShow s) <> (fromChar ':') <> (fromShow f)
     ab ma = case ma of
         (Just  a) -> fromShow a
         Nothing -> fromString "\"\""
     tab subs msg = spJoin $
-        [tb $ msgTime msg] ++ (subs msg) ++ [ab $ msgAttributee msg]
+        [tb $ duMsgTime msg] ++ (subs msg) ++ [ab $ duMsgAttributee msg]
```

Comment:
After an hour of review, I'm starting find it really hard to read all these terse variable names and keep a clue of what's going on in my head. We've got `tb`, `ab`, `ma`, `ib` `s`, `f`, `a`, `b`, `vb`, `vs`, `bs`, `i`, and `tab` at least going on this function alone! :tired_face: